### PR TITLE
feat: delete multiple photos

### PR DESCRIPTION
**Pull request by SamShanks1**
**Pull Request Description**

_Please include a general description of the changes made, why they were necessary, 
and any other general things to note._

**Pull Request Checklist**:
* [ ] Have you followed the guidelines in our Contributing document and Code of Conduct?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/project-error/npwd/pulls) for the same update/change?
* [ ] Have you built and tested NPWD in-game after the relevant change?

### CODE REVIEW

**Comment by SamShanks1** [comment]
#842

**Comment by SamShanks1** [comment]
#838

File: phone/src/apps/camera/components/NewPhotoButton.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import { useIsEditing } from '../hooks/state';
+import fetchNui from '@utils/fetchNui';
+import { ServerPromiseResp } from '@typings/common';
+import { GalleryPhoto, PhotoEvents } from '@typings/photo';
+import { usePhotoActions } from '../hooks/usePhotoActions';
+import { useCheckedPhotosValue } from '../hooks/state';
+import { useSnackbar } from '@os/snackbar/hooks/useSnackbar';
+import { useTranslation } from 'react-i18next';
+
+const useStyles = makeStyles((theme) => ({
```
Comment (itschip) [inline phone/src/apps/camera/components/NewPhotoButton.tsx:14]: `makeStyles` is deprecated. Use `styled` instead. 

Example:
```js
const FloatBtn = styled(Fab)({})
```

File: resources/client/cl_photo.ts [hunk 2]
```diff
@@ -155,3 +155,5 @@ const takePhoto = () =>
 RegisterNuiProxy(PhotoEvents.FETCH_PHOTOS);
 RegisterNuiProxy(PhotoEvents.DELETE_PHOTO);
 RegisterNuiProxy(PhotoEvents.SAVE_IMAGE);
+RegisterNuiProxy(PhotoEvents.DELETE_MULTIPLE_PHOTOS);
+RegisterNuiProxy(PhotoEvents.SAVE_IMAGE);
```
Comment (itschip) [inline resources/client/cl_photo.ts:159]: Looks like there's a duplicate of SAVE_IMAGE.

**Review by itschip — CHANGES_REQUESTED** [review]

**Pull request merged**